### PR TITLE
admin: support cleanup multi-valued index

### PR DESCRIPTION
**Pull request by xiongjiwei**
Signed-off-by: xiongjiwei <xiongjiwei1996@outlook.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/40427

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: executor/admin.go [hunk 1]
```diff
@@ -551,7 +551,7 @@ func (e *CleanupIndexExec) getIdxColTypes() []*types.FieldType {
 	}
 	e.idxColFieldTypes = make([]*types.FieldType, 0, len(e.columns))
 	for _, col := range e.columns {
-		e.idxColFieldTypes = append(e.idxColFieldTypes, &col.FieldType)
+		e.idxColFieldTypes = append(e.idxColFieldTypes, col.FieldType.ArrayType())
```
Comment (xhebox) [inline executor/admin.go:554]: Maybe shoud link a description to `index.go`,

Comment (xiongjiwei) [reply executor/admin.go:554]: ok

**Review by xhebox — APPROVED** [review]

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**